### DISCUSSION TITLE
Automatically update branch when out-of-date with base branch

### DISCUSSION THREAD

**Discussion by tonglil** [community/community#61993] [Pull Requests]
### Select Topic Area

Product Feedback

### Body

Instead of asking us to click `update branch`, provide a feature to automatically update branches/PRs (perhaps on a per-PR toggle) without having to come click on it.

<img width="645" alt="image" src="https://github.com/community/community/assets/3250776/c0c3f7a0-6e88-4a84-b023-734c9a2a29f3">

Related to https://github.com/orgs/community/discussions/17880

**Comment by laysauchoa** [comment]
I agree, what are people using instead?
